the_well/data/datasets.py

```
@@ -276,26 +276,12 @@ def __init__(
 
         # Initialize normalization classes if True
         if use_normalization and normalization_type:
-            try:
-                with self.fs.open(self.normalization_path, mode="r") as f:
-                    stats = yaml.safe_load(f)
+            with self.fs.open(self.normalization_path, mode="r") as f:
```

Remove the large try-except block to avoid silent failure.